app/routes/datasets/downloads.py

```diff
@@ -38,7 +38,7 @@ async def download_json(
     sql: str = Query(..., description="SQL query."),
     geostore_id: Optional[UUID] = Query(None, description="Geostore ID."),
     geostore_origin: GeostoreOrigin = Query(
-        GeostoreOrigin.gfw, description="Origin service of geostore ID."
+        GeostoreOrigin.gfw, description="Service to search first for geostore."
```

Comment:
Same as above as well - is this supposed to be part of this PR, or is this merged from a different PR?